lib/interceptor.js

```diff
@@ -54,6 +60,48 @@ var interceptor = {
 
     done(window[NAMESPACE]);
 
+    function replaceSendBeacon() {
+      var _sendBeacon = window.navigator.sendBeacon;
+      var interceptSendBeacon = function (url, payload) {
+        var request = {
+          method: 'POST',
+          requestHeaders: {},
+          requestBody: payload,
+          url: url,
+        };
+
+        addPendingRequest(request);
+
+        var isRequestQueuedForTransfer = _sendBeacon.apply(window.navigator, [
+          url,
+          payload,
+        ]);
+
+        completeSendBeaconRequest(request, {
+          body: isRequestQueuedForTransfer,
+        });
```

Comment:
The [sendBeacon() method](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon) returns true if the [user agent](https://developer.mozilla.org/en-US/docs/Glossary/User_agent) successfully queued the data for transfer. Otherwise, it returns false.

We probabaly want to examine if the request was queued or not. So body will be a boolean here